erdgo/blockchain/proxy_test.go

```diff
@@ -0,0 +1,73 @@
+package blockchain
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo/data"
+)
+
+func TestElrondProxy_GetNetworkEconomics(t *testing.T) {
+	t.Skip()
+
+	ep := NewElrondProxy("https://gateway.elrond.com")
```

Comment:
`http://localhost:8079` ?

Reply:
this test is skipped, can be left as it is and used only when necessary.

Reply:
done. also test was moved from this directory